soc/xtensa/nxp_adsp/imx8ulp/Kconfig.defconfig.series

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NXP_IMX8ULP
+
+config SOC_SERIES
+	string
+	default "imx8ulp"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_imx8ulp_adsp"
+
+config SOC
+	string
+	default "nxp_imx8ulp"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 528000000 if XTENSA_TIMER
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 50000
+
+config DYNAMIC_INTERRUPTS
```

Comment:
Should probably enable this at board level. Any more opinions on this?

Reply:
We should stick with the approach we had for previous enabled boards. The we can fix it later (tm).